### ISSUE TITLE
Discussion comment should get avatar from ENS (if exists)

### ISSUE THREAD

**Issue by giladt** [PrimeDAO/prime-deals-dapp#370]
Currently, all avatars for comments are generic robots. If some user has an ENS name that includes an avatar, it should be shown as the profile avatar.

**Comment by curvelabs-dev** [comment]
@blink-km @BartuCulha how important is this?

**Comment by blink-km** [comment]
Yes during the stand up, we decided to make the avatars optional.

**Comment by blink-km** [comment]
Sent to backlog
